.vscode/launch.json

```diff
@@ -24,8 +24,10 @@
       "request": "launch",
       "runtimeExecutable": "${execPath}",
       "args": [
-        "${workspaceFolder}/src/Razor/test/testapps/",
-        "--extensionDevelopmentPath=${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension"
+        "C:\\Users\\rybrande\\projects\\temp\\CleanComponents",
+        "--extensionDevelopmentPath=${workspaceFolder}/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension",
+        "--enable-proposed-api razor-vscode",
```

Comment:
I imagine we need to bring this into our VSCode functional test infrastructure so the registration of the proposed APIs doesn't throw don't we?

Reply:
So, here's the deal with Proposed API's. As per https://code.visualstudio.com/api/advanced-topics/using-proposed-api

> They are subject to change, only available in Insiders distribution and cannot be used in published extensions.

So, getting our Functional tests to run with these included would require making them use the insider build as well as marking them as proposed enabled, but more critically, merging this branch into master would mean **we could not ship until this API is no longer proposed, or we removed the feature**. That seems D A N G E R O U S, so what if instead of merging this into master we created a feature branch? Then everyone is safe from this madness, I can still make PR's against that branch to get reviews, and we don't have to worry about the functional tests (I'm not changing existing code, and don't have apis which would enable functional tests for this feature) until we merge back in.

Reply:
Lets talk more about options in person

Reply:
>  getting our Functional tests to run with these included would require making them use the insider build

We already use insiders build for our functional tests